specs/phase1/beacon-chain.md

```diff
@@ -557,26 +308,30 @@ def get_shard_committee(beacon_state: BeaconState, epoch: Epoch, shard: Shard) -
         indices=active_validator_indices,
         seed=seed,
         index=shard,
-        count=get_active_shard_count(beacon_state),
+        count=get_active_shard_count(beacon_state, epoch),
     )
 ```
 
-#### `get_light_client_committee`
+#### `compute_proposer_index`
```

Comment:
`get_committee_count_per_slot` is prefixed with `Updated` - it should be the same here,